frontend/src/scenes/experiments/Experiments.tsx

```diff
@@ -79,29 +83,56 @@ export function Experiments(): JSX.Element {
     return (
         <div>
             <PageHeader
-                title="Experiments"
-                style={{ borderBottom: '1px solid var(--border)', marginBottom: '1rem' }}
+                title={
+                    <div className="flex-center">
+                        Experimentation
+                        <LemonTag type="warning" style={{ marginLeft: 6, lineHeight: '1.4em' }}>
+                            BETA
+                        </LemonTag>
+                    </div>
+                }
+                style={{ borderBottom: '1px solid var(--border)', paddingBottom: '0.5rem', marginBottom: '1rem' }}
                 buttons={
-                    <LinkButton
-                        type="primary"
-                        data-attr="create-experiment"
-                        to={urls.experiment('new')}
-                        icon={<PlusOutlined />}
-                    >
-                        New Experiment
-                    </LinkButton>
+                    hasAvailableFeature(AvailableFeature.EXPERIMENTATION) ? (
+                        <LinkButton
+                            type="primary"
+                            data-attr="create-experiment"
+                            to={urls.experiment('new')}
+                            icon={<PlusOutlined />}
+                        >
+                            New Experiment
+                        </LinkButton>
+                    ) : undefined
                 }
             />
-            <LemonTable
-                dataSource={experiments}
-                columns={columns}
-                rowKey="id"
-                loading={experimentsLoading}
-                defaultSorting={{ columnKey: 'id', order: 1 }}
-                pagination={{ pageSize: 100 }}
-                nouns={['Experiment', 'Experiments']}
-                data-attr="experiment-table"
-            />
+            {hasAvailableFeature(AvailableFeature.EXPERIMENTATION) ? (
+                <LemonTable
+                    dataSource={experiments}
+                    columns={columns}
+                    rowKey="id"
+                    loading={experimentsLoading}
+                    defaultSorting={{ columnKey: 'id', order: 1 }}
+                    pagination={{ pageSize: 100 }}
+                    nouns={['Experiment', 'Experiments']}
+                    data-attr="experiment-table"
+                />
+            ) : (
+                <PayGatePage
+                    featureKey="experimentation"
+                    header={
+                        <>
+                            Introducing <span className="highlight">Experimentation</span>!
+                        </>
+                    }
+                    caption={
+                        <>
+                            Test changes to your product and how they impact your users. A/B test new features and
```

Comment:
@joethreepwood for 👀 on copy

Reply:
LGTM

Reply:
There's some repetition here. I also think 'improve your metrics' as a goal is a bit cynical compared to 'build better products' 

```suggestion
                            A/B test changes and new features to see how they impact your users and improve your product!
```

Or, the slightly more active

```suggestion
                            Improve your product by A/B testing new features to discover what works best for your users.
```